### ISSUE TITLE
Gantry 5.5.17 | Mobile Menu

### ISSUE THREAD

**Issue by Dave-Skye** [gantry/gantry5#3198]
The mobile menu toggle does not seem to be working after upgrading to the latest gantry version.

Anyone else having this problem?

Gantry 5.5.17 with Hydrogen.

**Comment by hexplor** [comment]
Can you provide a link to your website? Working on my install.

**Comment by N8Solutions** [comment]
@Dave-Skye it's working for me. I'm using the Hydrogen template. What template are you using?

**Comment by Hotrods007** [comment]
I see this to be the case only when adjusting the desktop view to mobile size...  There is no hamburger.  but On the Phone there is..

**Comment by hexplor** [comment]
Correct. I just noticed that as well.

**Comment by sdotter** [comment]
I experience same issue... Someone already found a fix?

**Comment by dagroupinc** [comment]
Will be fixed in a future release?

**Comment by paulinohio** [comment]
sdotter: What directory are you changed this in?

**Comment by hexplor** [comment]
This should be fixed in CI builds (generating right now). Please let me know if the fix works for you. Thank you so much for your feedback. 
Link here: https://gantry.org/downloads#ci-builds

**Comment by paulinohio** [comment]
Forgive my ignorance but do I just install the CI Builds like would any extension?

**Comment by N8Solutions** [comment]
> Forgive my ignorance but do I just install the CI Builds like would any extension?

@paulinohio,
Yes, that is exactly what you do. Just like you were installing an update manually for Gantry 5. However, you should always take a backup first, and preferably do this on a staging/test site if at all possible. This way if things do blow up you can always restore quickly and if its not that bad you can simply reinstall the most recent stable version which in this case is v5.5.17.

**Comment by paulinohio** [comment]
Even after the update, the menu does not appear for me when adjusting the desktop view to mobile size. After I resize the page, I have refresh it in order for the hamburger menu to appear. I've tried this in many browsers.

**Comment by kolomvosg** [comment]
I had the same problem at a Joomla 4.4.1 site Gantry 5.5.17 and both Hydrogen 5.5.17.
I download the CI builds from https://gantry.org/downloads#ci-builds as hexplor suggested and installed it (over the installed Gantry 5.5.17) at my Joomla site and it worked immediatly. So now when i modify the width of the browser then the mobile menu (three lines) appears.

**Comment by paulinohio** [comment]
I installed it again and it works fine. Thanks so much!
